### DISCUSSION TITLE
tsconfig 관련 오류

### DISCUSSION THREAD

**Discussion by dldnjswns31** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#39] [General]
![image](https://user-images.githubusercontent.com/71388830/200757134-1774b8d0-ce69-4029-be59-f4159b75f3bc.png)

관련된 오류가 발생하시는 분 있으신가요?

@types/babel__core 패키지 설치도 하고 tsconfig내 compilerOptions에 types도 추가해봤는데 계속 오류 뜨네용..
``
   "types": [
      "babel"
    ],
``

**Comment by yeondooo** [comment]
어떤 상황에서 발생한건가요? npm run start 만해도 오류가 생기나요?

**Reply by dldnjswns31** [reply]
vscode 내에 오류는 떠있는데 npm start 했을 때 서버는 잘 돌아갑니다.

react-icons 설치 후에 오류가 발생했던것 같기도..

**Reply by yeondooo** [reply]
[링크](https://velog.io/@chl4842/%ED%83%80%EC%9E%85%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8%EC%97%90%EC%84%9C-%EB%9D%BC%EC%9D%B4%EB%B8%8C%EB%9F%AC%EB%A6%AC-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0)
보면 ts는 라이브러리 설치할때도 타입을 지정해주는것도 있다고 하는데 이것 때문일까요..?

**Comment by jaeyeong815** [comment]
저는 오류 안뜹니당 같은 환경인데 왜그럴까요.. 흠

**Reply by jaeyeong815** [reply]
저도 아까 react-icons 설치 했는데 `npm install react-icons --save` 요렇게 설치했었어요

**Comment by yeondooo** [comment]
가까운것 같으면서도 너무 먼 타입스크립트네요...🥲

**Comment by dldnjswns31** [comment]
타입 지정 라이브러리 설치했는데도 저러네요 ㅎㅎㅎ...
일단 더 찾아봐야겠어요
